test/integration/testArrayBind.js

```diff
@@ -348,45 +358,45 @@ describe('Test Array Bind', function ()
             [
               "5489",
               "SAMPLE",
-              "{\"user\":.........}",
+              "{\"user\":{\"SAP\":\"KUKY003\",\"email\":\"THERESA.KUK@DISNEY.COM\"}",
```

Comment:
to avoid any imaginable complications, can we please use another , preferably fictive email addresses with fictive domain in the dataset ? something like email1@addr.es. , email2@addr.es, and so on.